### ISSUE TITLE
Bake connections not accepted as keyword argument

### ISSUE THREAD

**Issue by robertjoosten** [krathjen/studiolibrary#326]
Hello,

Just wanted to let you know that currently the bakeConnected functionality is broken. I believe this is because in the animitem.py save function the keyword argument 'bake' us used rather than 'bakeConnected'.

```
    def save(self, objects, sequencePath="", **kwargs):
        """
        Save the animation from the given objects to the item path.
        
        :type objects: list[str]
        :type sequencePath: str
        :type kwargs: dict
        """
        super(AnimItem, self).save(**kwargs)

        # Save the animation to the given path location on disc
        mutils.saveAnim(
            objects,
            self.path(),
            time=kwargs.get("frameRange"),
            fileType=kwargs.get("fileType"),
            iconPath=kwargs.get("thumbnail"),
            metadata={"description": kwargs.get("comment", "")},
            sequencePath=sequencePath,
            bakeConnected=kwargs.get("bake")    # < ------------
        )
```

PS: This also seems to be an issues between time and frameRange

Thank you,
Robert

**Comment by krathjen** [comment]
Hi,

Thanks for letting us know.

This should now be fixed in version 2.9.5

https://github.com/krathjen/studiolibrary

Cheers,

Kurt

**Comment by robertjoosten** [comment]
Thank you
